aiohttp/http_websocket.py

```diff
@@ -78,10 +88,11 @@ class WSMsgType(IntEnum):
 DEFAULT_LIMIT: Final[int] = 2 ** 16
 
 
-_WSMessageBase = collections.namedtuple("_WSMessageBase", ["type", "data", "extra"])
-
+class WSMessage(NamedTuple):
+    type: WSMsgType
+    data: Any
+    extra: str
 
-class WSMessage(_WSMessageBase):
```

Comment:
Good idea on dropping the intermediate class though, have updated my PR with that.